website/content/en/preview/AWS/launch-templates.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: "Launch Templates and Custom Images"
-linkTitle: "Launch Templates"
-weight: 80
+title: "Launch Templates and Custom Images (Deprecated)"
+linkTitle: "Launch Templates (Deprecated)"
+weight: 0
 description: >
   Create custom launch templates for Karpenter
 ---
+**Karpenter recommends using generated launch templates by specifying requirements in the [Provisioner and ProviderRef]({{<ref "./provisioning.md" >}}) natively.**
 
 By default, Karpenter generates launch templates with the following features:
 - [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) for nodes.
 - Encrypted EBS root volumes with the default (AWS managed) KMS key for nodes.
 
-If these features are not sufficient for your use case (customizing node image, customizing EBS KMS key, etc), you need a custom launch template.
+In addition, Karpenter also fills the generated launch templates with the values in the [referenced AWSNodeTemplate]({{<ref "./provisioning.md" >}}). In addition, Karpenter also fills the generated launch templates with specified [Custom AMIs and User Data]({{<ref "./user-data.md" >}}).
 
-Karpenter supports using custom launch templates.
+If these features are not sufficient for your use case (customizing node image, customizing EBS KMS key, etc), you need a custom launch template.

```

Comment:
```suggestion
If these features are not sufficient for your use case (customizing node image, customizing EBS KMS key, etc), you may need a custom launch template.

```

Can you also reference an AWS doc around writing a custom launch template?

Reply:
There's a CFN template at the bottom of this file, and steps that walk through how to create a one. I feel like that's sufficient, WDYT?

Reply:
Probably just do a self-link to the anchor in the file then. That line just feels like it needs a link since it's referencing a user action that we have a recommendation for